src/react-components/ui-root.js

```diff
@@ -273,7 +276,7 @@ class UIRoot extends Component {
     this.props.scene.addEventListener("stateremoved", this.onAframeStateChanged);
     this.props.scene.addEventListener("share_video_enabled", this.onShareVideoEnabled);
     this.props.scene.addEventListener("share_video_disabled", this.onShareVideoDisabled);
-    this.props.scene.addEventListener("exit", this.exit);
+    this.props.scene.addEventListener("exit", () => this.exit());
```

Comment:
Super minor since this is an exit event, but we won't be able to clean it up this way. Should just be bound in the constructoror when defined like most of the other functions in this class.

Reply:
This was actually to fix a bug with scene exit. The exit method is already bound like other methods, but it takes a parameter, so I didn't want the event argument being passed along. Good point about the clean-up though, this does the wrong thing. I'll fix it.